utils/db/challenges.js

```diff
@@ -3,6 +3,15 @@ import firebase from "../firebase/firebase";
 import { getCurrentValue } from "./helper";
 import { getTeam } from "./teams";
 
+export async function openChallenge(assignedChallengeId, userId) {
+  const openTime = firebase.database.ServerValue.TIMESTAMP;
+
+  firebase
```

Comment:
This is an `async` function with no `await` call inside it.

Do we not want to await the update call to firebase?

Reply:
Hm it probably doesn't need to be async on second read except maybe to implicitly return a Promise which isn't really used. Similar to the other comment where I think there should be a try/catch but not really necessary to await